extensions.json

```diff
@@ -881,7 +882,7 @@
   "ms-vscode.vscode-js-profile-table": {
     "repository": "https://github.com/microsoft/vscode-js-profile-visualizer",
     "location": "packages/vscode-js-profile-table",
-    "prepublish": "npm run compile"
+    "custom": "npm install && npm run compile && cd packages/vscode-js-profile-table && npx vsce package --yarn --out extension.vsix" 
```

Comment:
For the sake of readability and maintainability, I'd love to be able to optionally pass an array to `custom`.
In case of the `ms-vscode.vscode-js-profile-table`, your `custom` might look like this:

```json
"custom": [
    "npm install",
    "npm run compile",
    "cd packages/vscode-js-profile-table && npx vsce package --yarn --out extension.vsix",
    "pwd", "// Prints ${context.repo}"
]
```

This might be more readable especially when working with extensions which require a lot of commands being executed for building.

Reply:
Oh, that's a magnificent idea! It really would make it more readable and it would make total sense to do it. Will take a look this week / early next week.